decArgo_api/decoder_bindings/zip_nc_files.py

```diff
@@ -0,0 +1,40 @@
+"""Code to Zip decoded NC files."""
+
+import logging
+import os
+import traceback
+from io import BytesIO
+from pathlib import Path
+from zipfile import ZIP_DEFLATED, ZipFile
+
+
+class ZipNCFilesError(Exception):
+    """Raised when there is an issue zipping the NC files."""
+
+
+class ZipNCFiles:
+    """Methods to zip newly decoded NC files from the Coriolis Decoder."""
+
+    def __init__(self, wmonum: str):
+        """Initialise the instance with the wmonum and NC file location."""
+        self.wmonum = wmonum
+        self.base_output_location = Path(os.getenv("OUTPUT_LOCATION", f"/mnt/data/output/nc/{wmonum}")).resolve()
```

Comment:
I'm not wholly convinced that this is safe, especially when we have an API that could be serving concurrent requests. Would a UUID be safer?